packages/api/src/graphql/mutation.private.ts

```diff
@@ -261,6 +276,34 @@ export const GraphQLAdminMutation = new GraphQLObjectType<undefined, Context>({
       }
     },
 
+    sendJWTLogin: {
+      type: GraphQLNonNull(GraphQLString),
+      args: {
+        url: {type: GraphQLNonNull(GraphQLString)},
+        email: {type: GraphQLNonNull(GraphQLString)}
+      },
+      async resolve(
+        root,
+        {url, email},
+        {authenticate, dbAdapter, generateJWT, sendMailFromProvider}
+      ) {
+        const {roles} = authenticate()
+        authorise(CanSendJWTLogin, roles)
+
+        const user = await dbAdapter.user.getUser(email)
+        if (!user) throw new Error('User does not exist') // TODO: make this proper error
+        const token = generateJWT({userID: user.id})
+        const link = `${url}?jwt=${token}`
```

Comment:
I'm not sure if i understand what you mean. Who ever calls the GraphQL mutation provides the URL. Since we don't now where the login link needs to go.